uui/components/widgets/StatusIndicator.tsx

```diff
@@ -6,9 +6,13 @@ import css from './StatusIndicator.module.scss';
 export type StatusIndicatorColors = 'neutral' | 'info' | 'success' | 'warning' | 'critical';
 
 export interface StatusIndicatorProps extends IHasCX {
+    /** Component size. If omitted, 24 size will be used. */
     size?: '24' | '18' | '12';
+    /** Component color. */
     color?: StatusIndicatorColors;
+    /** Fill mode, solid is used by default */
     fill?: 'solid' | 'outline';
+    /** Component caption. It's required. */
```

Comment:
Remove 'It's required.' because it's clear from property definition.

Reply:
fixed